### ISSUE TITLE
export primary and secondary keys of azurerm_search_service

### ISSUE THREAD

**Issue by DimitriTchapmi** [terraform-providers/terraform-provider-azurerm#1873]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

When you build an azurerm_search_service with terraform, it just export you the id attribute. It will be great if we could get primary and secondary key in addition. This will be so usefull in many cases, for example when you need to automate tasks with Rest API. I'm trying to output those keys and use them in ansible tasks.

### New or Affected Resource(s)

* azurerm_search_service

### Potential Terraform Configuration

```hcl
output "search_pri_key" {
value = "${azurerm_search_service.mysearch.primary_key}"
}
```

**Comment by DimitriTchapmi** [comment]
For example
output "search_key" {
  value = "${azurerm_search_service.mysearch.primary_key}"
}

**Comment by tombuildsstuff** [comment]
hi @DimitriTchapmi 

Just to let you know that support for this shipped in v1.17 - you can upgrade to this by updating the version in the Provider block (as shown below) and then running `terraform init -upgrade`:

```
provider "azurerm" {
  version = "=1.17.0"
}
```

Thanks!

**Comment by DimitriTchapmi** [comment]
Thanks a lot
